app/modules/handler/containers/Prompt.js

```diff
@@ -95,8 +97,15 @@ class PromptContainer extends Component<Props> {
     } = prompt;
     if (!blockchain) return false;
     const loading = (system.EOSIOURI === 'PENDING' || system.EOSIOURIBUILD === 'PENDING');
-    const hasBroadcast = !!(response && (response.processed && response.processed.receipt.status === 'executed'));
-    const hasExpired = !!(prompt.tx && !hasBroadcast && Date.now() > Date.parse(`${prompt.tx.expiration}z`));
+    const hasBroadcast =
+      !!(response && (response.processed && response.processed.receipt.status === 'executed'));
+    const hasExpired =
+      !!(prompt.tx && !hasBroadcast && Date.now() > Date.parse(`${prompt.tx.expiration}z`));
+    const requestedActor = get(prompt, 'req.1.0.authorization.0.actor');
```

Comment:
This isn't a valid way to check for all types of transactions, it should probably be:

```
const requestedActor = get(prompt, 'tx.actions.0.authorization.0.actor');
```

The `req` itself can be either an action, an array of actions, or a full transactions. Try logging into bloks.io with an account you don't have loaded and try this PR, it won't work.

Reply:
I see, will make the change. Thanks!